frontend/src/components/ViewActivities/activity.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import * as utils from '../Utils/utils.js';
+import * as DBUTILS from '../../constants/dbconstants.js'
+import '../../styles/activities.css';
+
+class Activity extends React.Component {
+  /** @inheritdoc */
+  render() {
+    const activity = this.props.activity;
```

Comment:
Is it possible for this to be null?

Reply:
no, but validation would be nice. added.